### PR TITLE
Dynamic Framework target

### LINKED ISSUE #142: Add Carthage Support
Test the open PR that adds Carthage support.

### PR DESCRIPTION
**Pull request by heitortsergent**
This PR fixes #142. 

Test using Carthage (Obj-C/Swift):

- [x] Simulator
- [x] Device
- [x] Archive
- [x] Upload to App Store

### CODE REVIEW

**Comment by heitortsergent** [comment]
@terryhorner let me know how this looks (I made some changes to the README) and if it works for you. You can test it by installing Carthage, creating a new project, and using this in your Cartfile:

```
github "keenlabs/KeenClient-iOS" "framework"
```

File: README.md [hunk 1]
```diff
@@ -38,48 +41,19 @@ While the name of this repo implies that this SDK is strictly for iOS, it can al
 
 ### Installation
 
-Installing the client should be a breeze. If it's not, please let us know at [contact@keen.io](mailto:contact@keen.io)!
+Installing the client should be a breeze, and there're 3 different ways to do it: Carthage, CocoaPods, and Binary. If you run into any problems, please let us know by opening an issue on this repository or sending us an email at [contact@keen.io](mailto:contact@keen.io)!
```
Comment (terryhorner) [inline README.md]: "there're" is a bit awkward to read. How about "there are"?

**Comment by terryhorner** [comment]
:+1: other than my comment.

File: README.md (hunk 1 above)
Comment (heitortsergent) [inline README.md]: Done. :+1:

**Pull request merged**